_posts/2020-04-10-notes-from-community-call-2-april-2020.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Notes from community call - 2 April 2020"
+date: 2020-04-10
+author: Jan Ainali
+type: blogpost
+excerpt: Does the 'How to test?' section stand the test?
+categories:
+  - Community call
+---
+
+# 2 April 2020 community call
+
+## Attendees
+
+* [Jan](https://publiccode.net/team/jan-ainali.html) (chair)
+* [Alba](https://publiccode.net/team/alba-roza.html) (notes)
+* [Laura](https://publiccode.net/team/laura-scheske.html)
+* [Boris](https://publiccode.net/team/boris-van-hoytema.html)
+* [Eric](https://publiccode.net/team/eric-herman.html)
+
+## Update from the Foundation
+
+* We have launched a website for Signalen at https://signalen.org
+* [Laura](https://publiccode.net/team/laura-scheske.html) has joined the Foundation for Public Code as Membership developer
```

Comment:
Both of these are not about the standard, I think they are kind of lost here. But we did talk about them so they can be here.

Reply:
They are not, but they are inline with the upcoming changes that you have approved. See [the section Meeting preparations](https://github.com/publiccodenet/about/pull/583/files#diff-9b62c1e44831737ce8ace270c91e17aa). Should those be changed?

Reply:
Same.